build.sbt

```diff
@@ -293,46 +230,21 @@ val zonesFilterFn = (x: String) => x == "Europe/Helsinki" || x == "America/Santi
 lazy val demo = crossProject(JSPlatform, JVMPlatform, NativePlatform)
   .in(file("demo"))
   .dependsOn(core)
-  .enablePlugins(TzdbPlugin)
+  .enablePlugins(TzdbPlugin, NoPublishPlugin)
   .settings(
-    scalaVersion   := scalaVer,
-    name           := "demo",
-    publish / skip := true,
-    Keys.`package` := file(""),
-    zonesFilter    := zonesFilterFn,
-    dbVersion      := TzdbPlugin.Version(tzdbVersion)
+    name            := "demo",
+    Keys.`package`  := file(""),
+    zonesFilter     := zonesFilterFn,
+    dbVersion       := TzdbPlugin.Version(tzdbVersion),
+    tlFatalWarnings := false,
+    Test / test     := (Compile / run).toTask("").value
```

Comment:
What does it do?

Reply:
I'll add a comment. When you call `test` it invokes `run`. This is so we don't have to add a separate step to CI for checking that.